insertBaseData.sql

```diff
@@ -712,7 +712,7 @@ INSERT INTO approvalSteps (uuid, name, advisorOrganizationUuid)
 INSERT INTO approvers (approvalStepUuid, positionUuid)
 	VALUES ((SELECT uuid from approvalSteps where name = 'Default Approvers'), (SELECT uuid from positions where name = 'ANET Administrator'));
 
--- Set approvalStepUuid's from organizations with default
+-- Set approvalStepUuids from organizations with default
```

Comment:
Please remove this change from the PR, it is unrelated (and *'s* with apostrophe is generally used as the plural form throughout the comments).

Reply:
The reason I made this change is because the ` in a comment confuses the jdbi script parser. With it the script will not load.

Reply:
Hah, that would seem to me to be a bug in the parser (any character should be allowed in a comment, quotes needn't be balanced; it looks suspiciously like https://github.com/jdbi/jdbi/issues/75 ), but I'll give in .